src/test/java/tests/LoginPage.java

```diff
@@ -0,0 +1,26 @@
+package tests;
+
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import staticdata.WebUrl;
+
+public class LoginPage extends BaseTest {
+
+    private static final String USERNAME = "standard_user";
+    private static final String PASSWORD = "secret_sauce";
+
+    @Test
+    public void isKeyboardArrowsWorksTest() {
```

Comment:
@outthedj change please name of test method, seems like it's from another project